metadata/search/search.go

```diff
@@ -149,11 +152,21 @@ func (s Search) RunSearch(q string) ([]ResourceDoc, error) {
 	var searchResults []ResourceDoc
 
 	for _, hit := range results.Hits {
-		doc := hit.(map[string]interface{})
+				doc := hit.(map[string]interface{})
+
+		var tags []string
+		if tagSlice, ok := doc["Tags"].([]interface{}); ok {
+			for _, tag := range tagSlice {
+				if strTag, ok := tag.(string); ok {
+					tags = append(tags, strTag)
+				}
+			}
+		}
```

Comment:
I'm not certain but it looks like you might have some formatting issues in this file.

Reply:
Tried this instead- https://github.com/featureform/featureform/blob/c25645fa992f010c65d955ea66799b006099d574/metadata/search/search.go#L161